commit 6776c0c6ed61f4774bb079657ba06e9df302737e (refs/changes/42/5242/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-12-14T11:26:15-08:00 (10 months ago)
    
    TRITON-1043 VMAPI workflow function putVm should use job.params rather than job.params.payload

diff --git a/lib/workflows/job-common.js b/lib/workflows/job-common.js
index 5b9f560..4764c16 100644
--- a/lib/workflows/job-common.js
+++ b/lib/workflows/job-common.js
@@ -304,7 +304,7 @@ function pollTask(job, cb) {
 function putVm(job, cb) {
     var vmapi;
 
-    if (job.params.payload.do_not_inventory) {
+    if (job.params.do_not_inventory) {
         cb(null,
             'VM has do_not_inventory set, no need to putVm to VMAPI');
         return;
